recipe/deactivate-go-core.bat

```diff
@@ -0,0 +1,2 @@
+@set "GOROOT=%CONDA_GOROOT_BACKUP%"
```

Comment:
Where is the equivalent file for Linux/macOS? Has this been dropped deliberately?